test/bundler/bundler_edgecase.test.ts

```diff
@@ -2031,6 +2031,41 @@ describe("bundler", () => {
     },
   });
 
+  itBundled("edgecase/NoOutWithTwoFiles", {
+    files: {
+      "/entry.ts": `
+        import index from './index.html'
+        console.log(index);
+      `,
+      "/index.html": `
+        <head></head>
+      `,
+    },
+    generateOutput: false,
+    backend: "api",
+    onAfterApiBundle: build => {
+      expect(build.success).toEqual(true);
+      expect(build.outputs).toBeArrayOfSize(2);
```

Comment:
can you verify that the build.outputs items are correct? mainly if you read the outputted `index.html`, you get the text. maybe also check if it's content type is right.

Reply:
Makes sense, have done so in newest commit!